week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -16,14 +12,6 @@ const user: any = {
   },
 }
 
-function hasAddress(user: any): any {
-  return Boolean(user.address)
-}
-
-function hasGivenAge(requiredAge: any): any {
-  return (user: any): any => user.age >= requiredAge
-}
-
 const isAdult = hasGivenAge(18)
```

Comment:
To też powinno mieć wydzielone miejsce. Skoro przeniesione zostały te funkcje wyżej.